pxfish/operation_type.py

```diff
@@ -180,7 +180,8 @@ def push(*, session, path):
     parent_type_name = 'operation type'
     component_names = operation_type_code_names()
 
-# TODO: Change so it only pushes test file when testing
+    if test:
+        component_names = component_names[0:1]
```

Comment:
This depends on the code names to be in a particular order, and opens up an opportunity for a really hard to find bug.

Change `push` so that it takes `component_names` as an argument, and pass the component names to be pushed instead of the `test` flag. You could always make the assignment to component_names on line 181 conditional on the argument being None to avoid having to change the other calls.